File: www/node/test/p137_2.py
import matplotlib.pyplot as plt # プロットオブジェクトをインポート
riritsu = 0.05 # 利率
yokin = [100000] # 預金配列の最初の値は 10 万
for i in range(10): #i の値を 0 〜 9 まで、10 回繰り返す
 risoku = int(yokin[i]*riritsu) # 利息は現在の預金額×利率 ( 整数化 )
 yokin.append(yokin[i]+risoku) # 配列に計算結果を追加
plt.title("FUKURI KEISAN") # グラフのタイトル
plt.xlabel("Year") #X 軸のラベル
plt.ylabel("Yokin[YEN]") #Y 軸のラベル
plt.plot(yokin, marker="o") # グラフをプロット
plt.show() # プロットオブジェクトを表示

File: www/pyjstest/p137_2.js
define('__main__',function (require,exports,module) {
  var PYLIB=require('http://localhost/runtime/lib/python/PyLib.js');
  var __top=PYLIB.moduleScope(PYLIB, false);
  PYLIB.LoopChecker.reset();
  __top.plt=require('http://localhost/runtime/lib/python/py_matplotlib,pyplot.js').install(PYLIB);
  __top.riritsu=0.05;
  __top.yokin=[100000];
  for (__top.i of __top.range(10)) {
    PYLIB.LoopChecker.check();
    __top.risoku=__top.int((__top.yokin.__getitem__(__top.i)).__mul__(__top.riritsu));
    __top.yokin.__getattribute__('append')((__top.yokin.__getitem__(__top.i)).__add__(__top.risoku));
  }__top.plt.__getattribute__('title')("FUKURI KEISAN");
  __top.plt.__getattribute__('xlabel')("Year");
  __top.plt.__getattribute__('ylabel')("Yokin[YEN]");
  __top.plt.__getattribute__('plot')(__top.yokin,PYLIB.opt({marker:"o"}));
  __top.plt.__getattribute__('show')();
});
requirejs(['__main__'],function(){
            if (window.parent && window.parent.sendResult) {
                window.parent.sendResult($("#output").text(),"py");
            }});
